docs/applications/guidelines.md

```
@@ -0,0 +1,141 @@
+- Se seu app sincroniza dados entre a Nuvemshop e uma plataforma externa, ele deve garantir que todos os dados estejam consistentes.
+- Apps que não refletem mais a funcionalidade principal original submetida serão reavaliados e precisarão ser reenviados para uma nova revisão.
+
+## 4. Desempenho do App
+Para que os merchant sejam bem-sucedidos, suas lojas online devem ter velocidade e experiência de usuário de primeira classe.
+- Apps que afetam o frontend da loja (a vitrine) devem ter um impacto mínimo no desempenho.
+- Apps que usam webhooks para sincronizar dados precisam garantir que a sincronização seja eficiente e não sobrecarregue os recursos da loja.
+
+
+## 5. Anúncio do App na Loja
+O anúncio do seu app é o cartão de visitas para os lojistas. Ele precisa ser **claro, conciso e responder diretamente às perguntas** que um usuário em potencial possa ter. Para garantir que todas as informações essenciais estejam lá, destaque as seguintes seções obrigatórias no perfil do app:  
+
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)